pkg/commands/dashboard/item/create.go

```diff
@@ -0,0 +1,121 @@
+package item
+
+import (
+	"io"
+
+	"github.com/fastly/go-fastly/v9/fastly"
+
+	"github.com/fastly/cli/pkg/argparser"
+	"github.com/fastly/cli/pkg/commands/dashboard/common"
+	fsterr "github.com/fastly/cli/pkg/errors"
+	"github.com/fastly/cli/pkg/global"
+	"github.com/fastly/cli/pkg/text"
+)
+
+// NewCreateCommand returns a usable command registered under the parent.
+func NewCreateCommand(parent argparser.Registerer, globals *global.Data) *CreateCommand {
+	var c CreateCommand
+	c.CmdClause = parent.Command("create", "Create a custom dashboard item").Alias("add")
+	c.Globals = globals
+
+	// Required flags
+	c.CmdClause.Flag("dashboard-id", "Alphanumeric string identifying a Dashboard").Required().StringVar(&c.dashboardID)
+	c.CmdClause.Flag("title", "A human-readable title for the dashboard item").Required().StringVar(&c.title)
+	c.CmdClause.Flag("subtitle", "A human-readable subtitle for the dashboard item. Often a description of the visualization").Required().StringVar(&c.subtitle)
+	c.CmdClause.Flag("source-type", "The source of the data to display").Required().HintOptions(sourceTypes...).EnumVar(&c.sourceType, sourceTypes...)
+	c.CmdClause.Flag("metrics", "The metrics to visualize. Valid options depend on the selected data source (set flag once per Metric)").Required().StringsVar(&c.metrics)
```

Comment:
Since each instance of this flag add a single metric, I think it should be named `metric`, and the help text made singular as well. The final part of the help text can be something like "specify multiple times to include multiple metrics".

Same change can be made for `update`.